### PR TITLE
Use real assertions in the test suite

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

File: test/dust-test.js [hunk 1]
```diff
@@ -18,10 +19,13 @@ describe("bundalo dust bundler @dust@", function () {
 			if (err) {
 				return done(err);
 			}
-			if (data.greeting && bundloo.__cache()[path.normalize('nest/dusta.properties')] && !bundlee.__cache()[path.normalize('nest/dusta.properties')]) {
+			try {
+				assert.ok(data.greeting);
+				assert.ok(bundloo.__cache()[path.normalize('nest/dusta.properties')]);
+				assert.ok(!bundlee.__cache()[path.normalize('nest/dusta.properties')]);
```
Comment (grawk) [inline test/dust-test.js]: We don't necessarily need try/catch since a thrown exception in a test will function the same as calling `done(e)`

Comment (aredridel) [inline test/dust-test.js]: Except this isn't in the test callback, it's asynchronous from it. Which means if it's caught, it's via domains, and ew.

Comment (grawk) [inline test/dust-test.js]: Ugh. Learning on a Friday... Ok I know what a domain is now :) So does mocha explicitly make use of these?

Comment (aredridel) [inline test/dust-test.js]: Actually, looks like no, but uncaughtException handlers. Still, not a thing I'd want to fall back to.

**Comment by grawk** [comment]
:+1:

**Pull request merged**